src/components/hero/HeroSection.css

```diff
@@ -27,18 +30,33 @@
     background: #FFD500;
     border: 1px solid #000000;
   }
+
   #hero-title {
-    margin-left: 24px;
-    width: 1245px;
-    height: 128px;
+    margin-top: 48px;
+    margin-left: 10%;
+    width: 84vw;
     font-family: sans-serif;
-    font-size: 56px;
+    font-size: 3.7vw;
     font-style: normal;
     font-weight: 800;
-    line-height: 64px;
+    line-height: 114.286%; 
```

Comment:
Is this precision necessary?

Reply:
line-height precision copied from Figma

Reply:
I will approve this PR. But pls contact designers asking if we can make it an integer. Rendering line height with multiple decimal points desn't sound right to me. I suspect it will have performance impact too.